paris_rollins.py

```diff
@@ -8,14 +8,24 @@
 # can't use them because the M-Lab platform doesn't have them. This should
 # be revisited if the M-Lab platform is upgraded.
 
-# TODO(joshb): this version just provides functions to run paris-traceroute,
-# and manage a cache of recent IP addresses. Next step is to poll Web100 agent.
+# Needs to be run as root, with the Web100 libraries installed, in the NPAD
+# slice.
+#
+#   export PYTHONPATH=/home/iupui_npad/build/lib/python2.6/site-packages ; \
+#   export LD_LIBRARY_PATH=/home/iupui_npad/build/lib ; \
+#   ./paris_rollins.py
+#
+# TODO(joshb): this is for experimental use only. The next step is to replace
+# the old wrapper with this one.
 
-import os
 import multiprocessing
+import os
+import platform
```

Comment:
Consider regrouping so that native imports are first and third-party imports are after. I realize this project doesn't match our style guide, but if there's no consistent pattern, makes sense to use ours.